gtwrap/interface_parser/tokens.py

```diff
@@ -19,6 +21,18 @@
 
 LPAREN, RPAREN, LBRACE, RBRACE, COLON, SEMI_COLON = map(Suppress, "(){}:;")
 LOPBRACK, ROPBRACK, COMMA, EQUAL = map(Suppress, "<>,=")
+
+# Encapsulating type for numbers, and single and double quoted strings.
+# The pyparsing_common utilities ensure correct coversion to the corresponding type.
+# E.g. pyparsing_common.number will convert 3.1415 to a float type.
+_type = (pyparsing_common.number ^ QuotedString('"') ^ QuotedString("'"))
```

Comment:
`_type` is not a great name. Maybe `NUMBER_OR_QUOTED_STRING`?

Reply:
Renaming it to `NUMBER_OR_STRING`.